docs/02-app/01-building-your-application/07-configuring/05-mdx.mdx

```diff
@@ -40,7 +40,7 @@ npm install @next/mdx @mdx-js/loader @mdx-js/react @types/mdx
 
 <AppOnly>
 
-Create a `mdx-components.tsx` file at the root of your application (the parent folder of `app/` or `src/`):
+Create a `mdx-components.tsx` file at the root of your application (`src/` or the parent folder of `app/`):
```

Comment:
@Willem-Jaap I believe even with `/src`, the `mdx-components.tsx` file needs to be at the root of the application.

Reply:
The MDX docs state otherwise: [https://mdxjs.com/docs/getting-started/#nextjs](https://mdxjs.com/docs/getting-started/#nextjs)
Also the Next.js docs indicate that all application code can be placed under `src/` [(here)](https://nextjs.org/docs/app/building-your-application/configuring/src-directory)

It seems supported, example: (https://github.com/shuding/nextra/blob/main/packages/nextra-theme-docs/src/mdx-components.tsx)

Reply:
@Willem-Jaap Thank for the clarification!